protocol/avdl/keybase1/favorite.avdl

```diff
@@ -39,8 +63,7 @@ protocol favorite {
 
     union { null, Time } mtime;
 
-    // Is the local view of this folder hosed due to stuck CR?
-    FolderConflictType conflictType;
+    union { null, ConflictState } conflictState;
```

Comment:
Remove the definition of `FolderConflictType` above?  Or I guess not until the rest of the Go code is fixed up to not depend on it.

Reply:
Yeah, I think KBFSOps still returns that at this time.